maintenance/ManageInactiveWikis.php

```diff
@@ -203,13 +203,27 @@ private function handleClosedWiki(
 		}
 	}
 
-	private function notifyBureaucrats( string $dbname ): void {
+	private function notifyBureaucrats( string $dbname, RemoteWikiFactory $remoteWiki ): void {
+		$url = $remoteWiki->getServerName();
```

Comment:
Update this branch and use
```php
$validator = $this->getServiceContainer()->get( 'CreateWikiValidator' );
$url = $remoteWiki->getServerName() ?: $validator->getValidUrl( $dbname );
```
Which was added yesterday after this patch was opened.